studio/components/layouts/SQLEditorLayout/SQLEditorMenu.tsx

```diff
@@ -96,24 +98,18 @@ const SideBarContent = observer(() => {
         {IS_PLATFORM && (
           <div className="my-4 mx-3 space-y-1 px-3">
             <div className="flex items-center">
-              <Button
-                className={showCmdkHelper ? 'rounded-r-none px-3' : undefined}
-                block
-                icon={<IconPlus />}
-                type="default"
-                disabled={isLoading}
-                style={{ justifyContent: 'start' }}
-                onClick={() => handleNewQuery()}
-              >
-                New query
-              </Button>
-              {showCmdkHelper && (
-                <Dropdown
-                  align="end"
-                  side="bottom"
-                  sideOffset={3}
-                  className="max-w-[210px]"
-                  overlay={[
+              <Dropdown
```

Comment:
Styling is a little weird, can we replace the + with a chevron down as well to indicate its a dropdown
![image](https://github.com/supabase/supabase/assets/19742402/14dd21bd-ae90-4d6f-ace6-dd6102cf5e36)

Add separator too, font size seems too big
![image](https://github.com/supabase/supabase/assets/19742402/2ebbedbb-cf85-4e80-8df2-fec9e26ee813)